src/components/presentational/Chat/Chat.tsx

```diff
@@ -21,6 +21,8 @@ export interface ChatProps {
     onSendMessage: (newMessage: string) => void;
     loading?: string;
     inputDisabled?: boolean;
+    suggestions?: string[];
```

Comment:
Add a story for suggestions or include being able to update it in the default story

Reply:
Added.